vgaidarji/github-actions-pr-merger
# Fetch full PR object

### What has been done
1. Added additional method for fetching full PullRequest object information

### How to test
1. Merge PR
2. Leave a comment on PR to trigger the action build
3. Confirm PullRequest payload is printed to console
